--- common/ProtocolGroup.cc	2022-07-27 14:06:18.000000000 +0100
+++ common/updatedProtocolGroup.cc	2022-11-08 10:38:02.601337952 +0000
@@ -119,6 +119,7 @@
     { IP_PROT_SCTP, &Protocol::sctp },
     { IP_PROT_IPv6EXT_MOB, &Protocol::mobileipv6 },
     { IP_PROT_MANET, &Protocol::manet },
+    { IP_PROT_RDP, &Protocol::rdp },
 
     { IP_PROT_LINK_STATE_ROUTING, &Protocol::linkStateRouting },
     { IP_PROT_FLOODING, &Protocol::flooding },          // INET specific non-standard protocol
