--- common/ProtocolGroup.cc	2021-05-18 11:25:41.000000000 +0100
+++ common/updateProtocolGroup.cc	2022-11-05 22:50:19.261149384 +0000
@@ -116,7 +116,7 @@
     { 132, &Protocol::sctp },
     { 135, &Protocol::mobileipv6 },
     { 138, &Protocol::manet },
-
+    { 144, &Protocol::rdp },
     { 249, &Protocol::linkStateRouting },    // INET specific: Link State Routing Protocol
     { 250, &Protocol::flooding },    // INET specific: Probabilistic Network Protocol
     { 251, &Protocol::probabilistic },    // INET specific: Probabilistic Network Protocol
