--- common/Protocol.cc	2022-11-08 10:35:13.833072195 +0000
+++ common/updatedProtocol.cc	2022-11-08 10:38:56.910684105 +0000
@@ -138,6 +138,7 @@
 const Protocol Protocol::udp("udp", "UDP", Protocol::TransportLayer);
 const Protocol Protocol::xmac("xmac", "X-MAC");
 const Protocol Protocol::xtp("xtp", "XTP");
+const Protocol Protocol::rdp("rdp", "RDP", Protocol::TransportLayer);
 
 // INET specific conceptual protocol identifiers
 const Protocol Protocol::ackingMac("ackingmac", "Acking MAC");
